.github/workflows/ci-base-tests.yml

```diff
@@ -2,12 +2,10 @@ name: SMARTS CI
 
 on: [push]
 
-env:
-  venv_dir: .venv
```

Comment:
What about PRs that add a new package? Looks like a docker build will need to be deployed first...but a new package needs to go out before we'd deploy a new build...

Sounds like a 🐔 and 🥚

Reply:
That would be the case but we run an update to the packages already in in the scripts

https://github.com/huawei-noah/SMARTS/blob/master/.github/workflows/ci-base-tests.yml#L26-L29